common/zlib.cpp

```diff
@@ -494,6 +494,10 @@ class GZipWriteStream : public WriteStream {
 
 SeekableReadStream *wrapCompressedReadStream(SeekableReadStream *toBeWrapped, uint32 knownSize) {
 	if (toBeWrapped) {
+		if (toBeWrapped->eos() || toBeWrapped->err() || toBeWrapped->size() < 2) {
+			delete toBeWrapped;
+			return NULL;
```

Comment:
It would be preferable to use nullptr here, as this returns a pointer

Reply:
Indeed - I've use `NULL` as this was used a few lines below, too - will replace that one, too